index.js

```diff
@@ -468,6 +472,33 @@ class GoogleAuth extends DbBase {
       )
     })
   }
+  async resetAdminPassword(email, newPassword, passwordResetToken) {
+    assert.ok(this.conf.useDB, 'Cannot add admins if DB is not available')
+
+    assert.ok(typeof email === 'string', 'Email is required')
+    assert.ok(typeof newPassword === 'string', 'New Password is required')
+
+    const admin = await this._getAdmin(email)
+    if (!admin) throw new UserError('ADMIN_ACCOUNT_DOES_NOT_EXIST_OR_IS_NOT_ACTIVE')
+    if (admin.passwordResetToken !== passwordResetToken) throw new UserError('INVALID_passwordResetToken')
+    let expiryDate = new Date(admin.passwordResetSentAt)
```

Comment:
should be fine to use `const` here

```suggestion
    const expiryDate = new Date(admin.passwordResetSentAt)
```